developer-docs-site/docs/tutorials/run-a-local-testnet.md

```diff
@@ -64,34 +98,65 @@ We describe each method below.
       --server-url http://127.0.0.1:8080
 ```
 
-This will start a Faucet running locally without any restrictions to tokens that can be claimed / minted and make the serivce as accessible as the testnet started above. Faucets are stateless services that can be run in parallel.
+This will start a Faucet running locally without any restrictions to tokens that can be claimed and minted. This Faucet service will be as accessible as the testnet youy started above.
 
-### Using Docker
+## Using Docker
+
+This section describes how to start your local testing using Docker.
 
 1. Install [Docker](https://docs.docker.com/get-docker/) including [Docker-Compose](https://docs.docker.com/compose/install/).
-2. Create a directory for your local test validator network.
-3. Download the [validator testnet docker compose](https://github.com/aptos-labs/aptos-core/blob/main/docker/compose/validator-testnet/docker-compose.yaml) and [validator configuration](https://github.com/aptos-labs/aptos-core/blob/main/docker/compose/validator-testnet/validator_node_template.yaml).
-4. Start Docker-Compose `docker-compose up`
+2. Create a directory for your local test validator network, and `cd` into it.
+3. Download the YAML files for:
+
+  - [Validator testnet docker compose](https://github.com/aptos-labs/aptos-core/blob/main/docker/compose/validator-testnet/docker-compose.yaml) and
+  - [Validator configuration](https://github.com/aptos-labs/aptos-core/blob/main/docker/compose/validator-testnet/validator_node_template.yaml).
+
+4. Start Docker Compose by running the command:
+
+    ```
+    docker-compose up
+    ```
+
+### Example
+
+For example, the command sequence for the above steps 2 through 4 is shown below:
+
+  **NOTE**: If you do not have `wget` installed on your macOS, then run the below command to install the `wget` before you follow the below example.
 
-For simplicitly, you can perform 2 through 4 with the following commands:
+  ```
+  brew install wget
+  ```
+
+  After you installed `wget` proceed as below.
 
```

Comment:
Maybe we should consider using tables if we want to get detailed with platform specific commands. I think someone reviewing this should know how to install wget, so I'd prefer to leave it off. Especially since this is a Mac centric approach to doing this.

If we do want to go down this approach, we could consider tables for linux, windows, mac: https://docusaurus.io/docs/markdown-features/tabs

Reply:
Removed it.